core/src/avm2/globals/flash/ui/ContextMenuBuiltInItems.as

```diff
@@ -1,13 +1,97 @@
 package flash.ui
 {
-	public final class ContextMenuBuiltInItems {
-		public var forwardAndBack:Boolean = true;
-		public var loop:Boolean = true;
-		public var play:Boolean = true;
-		public var print:Boolean = true;
-		public var quality:Boolean = true;
-		public var rewind:Boolean = true;
-		public var save:Boolean = true;
-		public var zoom:Boolean = true;
-	}
+    public final class ContextMenuBuiltInItems {
+        private var _forwardAndBack:Boolean = true;
+        private var _loop:Boolean = true;
+        private var _play:Boolean = true;
+        private var _print:Boolean = true;
+        private var _quality:Boolean = true;
+        private var _rewind:Boolean = true;
+        private var _save:Boolean = true;
+        private var _zoom:Boolean = true;
+        
+        public function get forwardAndBack():Boolean {
+            return this._forwardAndBack;
+        }
+        
+        public function set forwardAndBack(value:Boolean):void {
+            this._forwardAndBack = value;
+        }
```

Comment:
Why did you convert from public fields to trivial getter/setters? Here and elsewhere

Reply:
This matches FP. I know it's kinda weird, but there's no harm in doing it, and this way, `describeType` output will match as well.